packages/playground/src/weblets/profile_manager.vue

```diff
@@ -801,7 +808,15 @@ async function login() {
       const keypairType = credentials.keypairTypeHash
         ? cryptr.decrypt(credentials.keypairTypeHash)
         : KeypairType.sr25519;
+
       await activate(mnemonic, keypairType as KeypairType);
+      const grid = await getGrid({ mnemonic: mnemonic, keypairType: keypairType as KeypairType });
+
+      const email = await getMetadata(grid!);
+      if (!email) {
+        createCustomToast("Email is Missing! Please enter your Email.", ToastType.warning);
+        router.push({ path: "/tf-chain/your-profile" });
```

Comment:
The metadata does not contain only the email, it may contain the SSH Keys also, I think you have to check it yourself

You may need to use the `readEmail` to do that